### PR TITLE
Add new TDD 101 exercise, plus minor challenge tweak

### PR DESCRIPTION
**Pull request by JoeShep**
We needed a simple TDD exercise to slot between writing tests for existing code (diceroll) and the beast that is Bag-o-Loot. This gives students a gentle intro to true Red, Green, Refactor by asking them to start with no code and a single test and end with a working calculator app.

Also, minor typo fix to a challenge exercise.

### CODE REVIEW

**Comment by gregkorte** [comment]
Is this the place where we should add the links for setting up mocha and chai?

https://www.sitepoint.com/unit-test-javascript-mocha-chai/#setting-up-the-tools
Chai cheat sheet:   http://ricostacruz.com/cheatsheets/chai.html

**Comment by JoeShep** [comment]
Yeah, good idea

File: orientation/resources/16-tdd-first-steps.md [hunk 1]
```diff
@@ -19,3 +19,19 @@ describe('add', () => {
...
+
+## Resources
+
+### package.json
+You will need to run `npm init -y` to create a package.json. Then run `npm install --save-dev mocha chai`
+
+Also, add the following to your package.json:
+```js
+  "scripts": {
+    "test": "mocha -R list --watch --recursive"
```
Comment (gregkorte) [inline orientation/resources/16-tdd-first-steps.md:31]: May want to prompt Windows users to add the flag `--colors`. I didn't get colors on git bash until I did.

**Comment by gregkorte** [comment]
Also, you may want to add the link in the additional resources section, so if they want to learn more about mocha installation such as running tests in the browser. Otherwise :+1:

**Pull request merged**